bot/commands/data.js

```diff
@@ -3,72 +3,179 @@ import { currentSeason } from "../globals.js";
+    ),
+  async execute(interaction) {
+    switch (interaction.options.getSubcommand()) {
+      case "scout":
+        await scoutPlayer(interaction);
+        break;
+      case "team":
+        await getTeamData(interaction);
+        break;
     }
-}
+  },
+};
 
-async function scoutPlayer(interaction) {
-    async function dataCollector(interaction) {
-        const player = interaction.options.getUser('player');
-        const startSeason = interaction.options.getNumber('from_season') || 0;
-        const endSeason = interaction.options.getNumber('through_season') || currentSeason.number;
+async function getTeamData(interaction) {
+  async function dataCollector(interaction) {
+    const team = interaction.options.getRole("team");
 
-        const replaysByWeek = await loadReplays(startSeason, endSeason, player.id);
-        const replays = replaysByWeek.flatMap(week => [week.game1, week.game2, week.game3, week.game4, week.game5]).filter(game => !!game);
+    const playerData = await loadTeamData(team.id);
 
-        return { playerSnowflake: player.id, startSeason, endSeason, replays };
-    }
+    return { team, playerData };
+  }
 
-    function verifier(data) {
-        const { playerSnowflake, startSeason, endSeason, replays } = data;
-        let failures = [];
+  function verifier(data) {
+    const { team, playerData } = data;
+    let failures = [];
+
+    if (playerData.length === 0) {
+      failures.push(
+        `No data found for ${userMention(team.id)} from the current season.`
+      );
+    }
 
-        if (startSeason > endSeason) {
-            failures.push(`You asked for replays ${italic('after')} season ${startSeason} but ${italic('before')} season ${endSeason}. Obviously there are none.`);
-        }
+    return failures;
+  }
 
-        if (replays.length === 0) {
-            failures.push(`No replays found for ${userMention(playerSnowflake)} between seasons ${startSeason} and ${endSeason}.`)
-        }
+  function responseWriter(data) {
+    const { team, playerData } = data;
 
-        return failures;
+    function makePlayerIntoRow(p) {
+      return p.name
+        .padEnd(29, " ")
+        .concat("|     ")
+        .concat(p.wins)
+        .concat("|         ")
+        .concat(p.act_wins)
+        .concat("|       ")
+        .concat(p.losses)
+        .concat("|       ")
+        .concat(p.act_losses)
+        .concat("|         ")
+        .concat(p.ties)
+        .concat("|")
+        .concat(p.star_points.toString().padStart(13, " "))
+        .concat("|   ")
+        .concat(Math.ceil(100 * p.stars) / 100);
     }
 
-    function responseWriter(data) {
-        const { playerSnowflake, startSeason, endSeason, replays } = data;
+    return `Data found for ${userMention(
+      team.id
+    )}:\n\`\`\`Name                         | Wins | Act Wins | Losses | Act Losses | Ties | Star Points | Stars\n`
+      .concat(playerData.map((p) => makePlayerIntoRow(p)).join("\n"))
+      .concat("```");
```

Comment:
```suggestion
    return `Data found for ${roleMention(team.id)}:\n`.concat(
      codeBlock(''.concat(
        'Name                         | Wins | Act Wins | Losses | Act Losses | Ties | Star Points | Stars\n`,
        '-----------------------------|------|----------|--------|------------|------|-------------|------',
        playerData.map((p) => makePlayerIntoRow(p)).join("\n")
      )
    );
```

Again, copying my formatting from postStandings in season.js